ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,438 @@
+
+When designing an EVPN Layer 3 Leaf Spine (L3LS) fabric you don't need to be an expert on Arista Design and Deployment Guides.
+All those rules and guidelines are integrated in the Ansible collection for Arista Validated Designs (AVD), or in short: Ansible AVD.
+
+Ansible AVD is an opinionated collection, meaning that field experiences and best practices from actual deployments at Arista customers are included.
+With Ansible AVD, users get up and running quickly without having to invent every part of the network configuration from scratch.
+This means only the most basic information must be provided by the user to get a new fabric up and running.
+ 
+Parameters which need to be defined by the user include:
+- Username/password for network devices and Cloudvision
+- OOB management IP range
+- IP scope for link addresses
+- IP scope for loopback interfaces
+- VLAN and VNI ranges
+- Device hostnames
+- Which devices are acting in each role, such as spine or leaf.
+
+All are details that must be decided upon in any fabric design. 
+
+In a standard spine-leaf topology, Ansible AVD takes care of all the work generating the full EOS configurations past this high level description of names and numbering for the network.
+
+## When to use Ansible AVD and when not to
+It’s important to mention when to, and perhaps more importantly, when not to use Ansible AVD.
+
+Automating provisioning of a network infrastructure makes most sense when the network is designed and built using clearly defined,
+repeatable patterns. It makes less sense when a network infrastructure is using a non-uniform design.
+Although you can use Ansible AVD to provision a non-uniform network while harvesting some benefits,
+you’re creating a lot of manual work and customization of Ansible AVD to achieve what you need,
+which may ultimately be more work than manually managing the network when every device is its own special case.
+
+## Change your mindset and culture
+Many network engineers prefer to use the CLI for all things networking, be it troubleshooting, ad-hoc operational configuration changes
+or full blown greenfield provisioning tasks.
+
+To harvest the full benefit of Ansible and Ansible AVD,
+you must learn to think differently and change your ways of working.
+
+Ansible should be your first choice when changing your network configuration,
+be it small or big. The CLI should only be used for troubleshooting i.e. viewing the state of your network,
+or those urgent scenarios at 2am where you need to change something very specific on just one or two devices
+followed by proper configuration cleanup later.
+
+## How does Ansible work
+Before we can tell Ansible to do anything, Ansible needs to know about the devices (or hosts as Ansible calls it) it handles.
+
+This is defined in the Ansible ***inventory*** or actually inventory.yml, which is another text file in YAML format,
```

Comment:
as per [Ansible docs](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#inventory-basics-formats-hosts-and-groups) the inventory can actually take other formats and it's not tied to yml only. We just happend to use yaml as the default for AVD